docs/document/docs/components/desktop/table.md

```diff
@@ -6,5 +6,126 @@ original_id: table
+| requiredIcon | boolean | false  | Show/Hide the required icon | |
+| render | function <br> `function(cellData, rowData, rowIndex) {}` | - | Renderer of the table cell. The return value should be a DOM. | <ul><li>3 params below provide more information for render function:</li> <ul><li>`cellData` is data of current cell rendered.</li><li>`rowData` is data of current row rendered.</li><li>`rowIndex` is index of current row rendered.</li></ul><li>If render function is not specified, the cell will display with default text.</li></ul>|
+
+
+### Event
+
+Here is a list of events that can be specified:
+
+| Name | Type | Description | Remark |
+| :--- | :--- | :--- | :--- |
+| change | function | The function fire when the table change. | `function(event: Event => void)` <br><br> - Take the event object of Event as the argument<br> - Can receive the following values when used `event.detail` <ul><li>`change-cell` (Triggered if the data in cell is changed)</li><ul><li>event.detail.`type` : change-cell</li><li>event.detail.`rowIndex`: the row's index has changed</li><li>event.detail.`data`: all data of table</li><li>event.detail.`oldData`: old data of table</li><li>event.detail.`field`: The columns has changed</li></ul><li>`add-row` (Triggered if button add row is clicked)</li><ul><li>event.detail.`type` : add-row</li><li>event.detail.`rowIndex`: The row’s index added.</li><li>event.detail.`data`: all data of table</li><li>event.detail.`oldData`: old data of table</li></ul><li>`remove-row` (Triggered if button remove row is clicked</li><ul><li>event.detail.`type` : remove-row</li><li>event.detail.`rowIndex`: The row’s index removed.</li><li>event.detail.`data`: all data of table</li><li>event.detail.`oldData`: old data of table</li></ul></ul>|
+
+### Constructor
+
+Table(options)<br>
+Here is a list of available constructors:
+
+#### Parameter
+
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| options | object | {} | Object that includes component properties | |
+
+---
+
+## Sample Code
+
+> Please check the [package installation](../../getting-started/quick-start.md#installation) method first.
+
+Here is a sample code when all parameters are specified:
+
+```javascript
+const Kuc = Kucs['1.x.x'];
+
+const header = kintone.app.getHeaderMenuSpaceElement();
+
+const renderAge = (dataCell) => {
+  const spanElement = document.createElement("span");
+  spanElement.innerText = `The age is ${dataCell}`;
+  return spanElement;
+};
+
+const renderName = (cellData) => {
+  const dropdown = new Dropdown({
+    items: [...],
```

Comment:
Please change from  [...] to
items: [
        {
          label: "John Brown",
          value: "John Brown",
        },
        {
          label: "Steven Gerrard",
          value: "Steven Gerrard",
        },
      ],

Reply:
I updated